packages/text-editor/src/components/CollaboratorEditor.svelte

```diff
@@ -79,6 +79,8 @@
 
   export let field: string | undefined = undefined
 
+  export let autoOverflow = false
```

Comment:
I don't see any place where we pass this

Reply:
We will pass it from qms :-)